Downgrade flex 2.6.4 to 2.6.3 to make it compile with GCC 7.3.0-16ubuntu3 (Ubuntu 18.04.1)

./stage1flex   -o stage1scan.c ./scan.l
Makefile:1701: recipe for target 'stage1scan.c' failed
make[5]: *** [stage1scan.c] Segmentation fault (core dumped)



diff -Naur buildroot-2017.08.x/package/flex/flex.hash buildroot-a2493cae0981867879af1c50ffee9ae01322986e/package/flex/flex.hash
--- buildroot-2017.08.x/package/flex/flex.hash	2018-10-16 01:07:12.258392656 -0400
+++ buildroot-a2493cae0981867879af1c50ffee9ae01322986e/package/flex/flex.hash	2018-10-16 00:59:12.094688046 -0400
@@ -1,2 +1,3 @@
 # Locally computed:
 sha256 e87aae032bf07c26f85ac0ed3250998c37621d95f8bd748b31f15b33c45ee995  flex-2.6.4.tar.gz
+sha256 68b2742233e747c462f781462a2a1e299dc6207401dac8f0bbb316f48565c2aa  flex-2.6.3.tar.gz
diff -Naur buildroot-2017.08.x/package/flex/flex.mk buildroot-a2493cae0981867879af1c50ffee9ae01322986e/package/flex/flex.mk
--- buildroot-2017.08.x/package/flex/flex.mk	2018-10-16 01:07:12.258392656 -0400
+++ buildroot-a2493cae0981867879af1c50ffee9ae01322986e/package/flex/flex.mk	2018-10-16 00:52:20.270285441 -0400
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-FLEX_VERSION = 2.6.4
+FLEX_VERSION = 2.6.3
 FLEX_SITE = https://github.com/westes/flex/files/981163
 FLEX_INSTALL_STAGING = YES
 FLEX_LICENSE = FLEX
